mandatory/2-function-creation.js

```diff
@@ -4,7 +4,15 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  if (userAge>=18 && isLoggedIn===true){
+    return true;
+  }else if (userAge>18 && isLoggedIn===true){
+  return false;
+}else {
+  return false;
+  }
+}
```

Comment:
The condition to accept the user or not is determining whether userAge >= 18 and if the user is logged in

`userAge >= 18`  -> this statement should return a boolean, so if you assign it to a variable, that variable's type should be a boolean. You can see the example code to see this here:

```js

let isUserAgeBigger = userAge >= 18;
console.log(typeof isUserAgeBigger); // this should be a boolean
console.log(isUserAgeBigger) // this should be the value of it
```

the variable isLoggedIn is already of type boolean, as you can see in the examples from the tests [here](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week2/pull/451/files?diff=split&w=0#diff-6b32acd5b3a1f33e1f035a68477c6641df187cb5ee57e41615bb942d518ade86L52)


Since you only want a boolean return from this function, you could just evaluate the boolean expression, by returning the statement inside the first `if` condition, since it will only return true if the statement inside the if condition (`userAge>=18 && isLoggedIn===true`) is true. 

So you could just do: 
```js
function isAcceptableUser(userAge, isLoggedIn) {
    return userAge>=18 && isLoggedIn===true
}
```

and it should return the behaviour above is doing. 

Also since `isLoggedIn` is a boolean value, you do not need to check if it's equal to true or not. you should be able to directly use it in the conditions.